tests/storage/cdi_config/conftest.py

```
@@ -0,0 +1,34 @@
+"""
+Pytest conftest file for CNV CDI Config tests
+"""
+
+import pytest
+from ocp_resources.cdi import CDI
+
+from utilities.hco import ResourceEditorValidateHCOReconcile
+from utilities.storage import cdi_feature_gate_list_with_added_feature
+
+
+@pytest.fixture()
```

Ruff (0.15.9)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)